### PR TITLE
Revamp of design aspect of the table with responsiveness table

### PR DESCRIPTION
**Pull request by ishan-me**
The following issues has been resolved

- [x] Need a visual makeover and UX improvements.
- [x] Ability to add logo

---

Leaderboard_UI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 OSS Leaderboard shows the top contributors based on their contribution activities on GitHub.
 
-<img src = "https://i.imgur.com/SPMKoJt.png" width = "700">
+<img src = "https://imgur.com/o32dq3s" width = "700">
```
Comment (yankeexe) [inline README.md:5]: Image link is broken, can you replace it with https://i.imgur.com/o32dq3s.png

File: README.md [hunk 2]
```diff
@@ -46,6 +46,13 @@ $ source .venv/bin/activate
 $ make setup
 ```
 
+4. Running docker locally inside your shell 
+
+```bash
+$ sh run-local.sh
+
+```
+
```
Comment (yankeexe) [inline README.md:55]: There's a section on README for running the script. 
Reference: 
https://github.com/leapfrogtechnology/oss-leaderboard#building

We can add more description here.

File: final_html_output.py [hunk 3]
```diff
@@ -11,14 +11,26 @@ def final_html_output(df: pd.DataFrame) -> str:
...
-        <body class="container">
-            <h1>OPENSOURCE LEADERBOARD</h1>
-            {table}
-        </body>
-    </html>
+  <html>
+    <head>
+      <title>Opensource Leaderboard Leapforg</title>
+    </head>
+    <link rel="stylesheet" type="text/css" href="../style.css" />
```
Comment (yankeexe) [inline final_html_output.py:18]: We can put the stylesheets inside the `assets` directory.

File: style.css [hunk 4]
```diff
@@ -1,69 +1,138 @@
...
+  margin-bottom: 20px;
+}
+
+td {
+  color: var(--white-color);
+}
+
+tbody tr:hover {
+  cursor: pointer;
+  background-color: #ffffff1f !important;
```
Comment (yankeexe) [inline style.css:79]: Can it be done without using `!important`?

Comment (ishan-me) [reply style.css]: Ok I will check on this one, just wanted to override previous ones.

File: final_html_output.py (hunk 3 above)
Comment (ishan-me) [reply final_html_output.py]: Ok sure

File: README.md (hunk 2 above)
Comment (ishan-me) [reply README.md]: Yup

File: README.md (hunk 1 above)
Comment (ishan-me) [reply README.md:5]: Fixed !

File: README.md [hunk 5]
```diff
@@ -17,6 +17,7 @@ $ cp .env.example .env
 #### Building
 
 ```bash
+$ sh run-local.sh
```
Comment (kabirbaidhya) [inline README.md:20]: This isn't needed, `make run` already does it.

Comment (ishan-me) [reply README.md:20]: ok thankyou

File: Pipfile [hunk 6]
```diff
@@ -0,0 +1,12 @@
+[[source]]
```
Comment (kabirbaidhya) [inline Pipfile:1]: Let's remove `Pipefile` & its lock file as we won't be using them. We're using pip + setuptools + virtualenv so far and might move towards poetry later.

All the dependencies are centralized here already - https://github.com/leapfrogtechnology/oss-leaderboard/blob/master/setup.py.

File: Pipfile.lock [hunk 7]
```diff
@@ -0,0 +1,93 @@
+{
```
Comment (kabirbaidhya) [inline Pipfile.lock:1]: Let's remove this too.

**Comment by kabirbaidhya** [comment]
@ishan-me Can you add a preview screenshot of how the UI looks like? Let's attach it to the PR description.

File: final_html_output.py [hunk 8]
```diff
@@ -11,14 +11,26 @@ def final_html_output(df: pd.DataFrame) -> str:
         html string
     """
     html_string = """
```
Comment (kabirbaidhya) [inline final_html_output.py:13]: Can you move the html from here to a separate html file - eg: `assets/index.html`. This way you can format and update it   properly.

And you can read the file here as string the same way.

File: README.md (hunk 1 above)
Comment (mesaugat) [reply README.md:5]: Can we use real names here? 

Let's use this as the URL `https://i.imgur.com/o32dq3s.png`

File: final_html_output.py [hunk 9]
```diff
@@ -11,14 +11,26 @@ def final_html_output(df: pd.DataFrame) -> str:
...
+      <div class="leaderboard">
+        <img
+        class="leaderboard__img"
+        src="../assets/LF_Opensource.svg"
+        alt="Leapfrog Opensource Logo"
+        />
+        <h1>Frogtoberfest Leaderboard</h1>
+        <div class="leaderboard__table">
+          {table}
+          </section>
```
Comment (mesaugat) [inline final_html_output.py:29]: Where does the section start though?

File: final_html_output.py [hunk 10]
```diff
@@ -11,14 +11,26 @@ def final_html_output(df: pd.DataFrame) -> str:
...
-        <head><title>Opensource Leaderboard</title></head>
-        <link rel="stylesheet" type="text/css" href="style.css"/>
-        <body class="container">
-            <h1>OPENSOURCE LEADERBOARD</h1>
-            {table}
-        </body>
-    </html>
+  <html>
+    <head>
+      <title>Opensource Leaderboard Leapforg</title>
```
Comment (kabirbaidhya) [inline final_html_output.py:16]:
```suggestion
      <title>Leapfrog Opensource Leaderboard</title>
```

File: final_html_output.py [hunk 11]
```diff
@@ -11,14 +11,26 @@ def final_html_output(df: pd.DataFrame) -> str:
...
+    </head>
+    <link rel="stylesheet" type="text/css" href="../assets/style.css" />
+    <body>
+      <div class="leaderboard">
+        <img
+        class="leaderboard__img"
+        src="../assets/LF_Opensource.svg"
+        alt="Leapfrog Opensource Logo"
```
Comment (kabirbaidhya) [inline final_html_output.py:24]:
```suggestion
        alt="Leapfrog Opensource"
```

File: README.md (hunk 1 above)
Comment (ishan-me) [reply README.md:5]: ok

File: final_html_output.py (hunk 9 above)
Comment (ishan-me) [reply final_html_output.py:29]: Yes its Unnecessary!

**Comment by ishan-me** [comment]
Addressed and worked on the comments received. 

UI

**Comment by kabirbaidhya** [comment]
The CI [build](https://travis-ci.com/github/leapfrogtechnology/oss-leaderboard/builds/188323648) is failing. Can you run `make format` and push the changes?

File: assets/index.html [hunk 12]
```diff
@@ -0,0 +1,17 @@
+<html>
+  <head>
+    <title>Opensource Leaderboard Leapforg</title>
```
Comment (kabirbaidhya) [inline assets/index.html:3]:
```suggestion
    <title>Leapfrog Opensource Leaderboard</title>
```

**Comment by kabirbaidhya** [comment]
Can we change this title? I think we can do `Frogtoberfest Leaderboard`.
image

**Comment by ishan-me** [comment]
Added the changes as suggested.

Leaderboard_UI

**Review by kabirbaidhya — APPROVED** [review]
Awesome. Looks good @ishan-me :100:

File: README.md [hunk 13]
```diff
@@ -2,7 +2,7 @@
 
 OSS Leaderboard shows the top contributors based on their contribution activities on GitHub.
 
-<img src = "https://i.imgur.com/SPMKoJt.png" width = "700">
+<img src = "https://imgur.com/lxiff4i" width = "700">
```
Comment (kabirbaidhya) [inline README.md:5]: One more thing - let's update the image to reflect the latest design.

**Review by yankeexe — APPROVED** [review]

File: README.md (hunk 13 above)
Comment (ishan-me) [reply README.md]: I have updates the link as well will validate it as well.

**Review by ishan-me — COMMENTED** [review]
Replaced workable image URL

**Review by bhattaraib58 — APPROVED** [review]

**Review by aanchal-adk — APPROVED** [review]

**Comment by mesaugat** [comment]
@ishan-me Can we change `Person_name` to real names? It could be anyone.

**Comment by ishan-me** [comment]
I don't want to be bias but I will set up a random name 😄

**Review by mesaugat — APPROVED** [review]
Looks good.

**Pull request merged**